### ISSUE TITLE
Crash on Startup

### ISSUE THREAD

**Issue by Miroslav008** [MissingDeadlines/iolite#30]
Hello! My IOLITE engine does not start because it crashes.
The CPU and GPU meet the minimum system requirements, even the optimal ones. My computer components:

Video card: GeForce GTX 1080 Ti
Processor: Intel Core i7-8700K
RAM size: 16 GB
OS: Windows 10

The latest drivers are installed (Vulkan is built-in, it seems).
Screenshot:

![image](https://github.com/MissingDeadlines/iolite/assets/129593085/ac9523e2-a053-4255-94ab-573bd9acbbb3)

**Comment by begla** [comment]
@Miroslav008 The `application.log` log file can be found in `Documents/IOLITE`. It would be great if you could post it here. Thanks!

**Comment by Miroslav008** [comment]
> @Miroslav008 The `application.log` log file can be found in `Documents/IOLITE`. Please post it here (alternatively via a direct message on Discord). Thanks!

```
INFO    (2024-02-06 16:04:55): Executing compatibility checks...
INFO    (2024-02-06 16:04:55):  CPU Vendor: GenuineIntel
INFO    (2024-02-06 16:04:55):  CPU Brand : Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz
INFO    (2024-02-06 16:04:55):  Checking required CPU features...
INFO    (2024-02-06 16:04:55):   SSE [Supported]
INFO    (2024-02-06 16:04:55):   SSE2 [Supported]
INFO    (2024-02-06 16:04:55):   SSE3 [Supported]
INFO    (2024-02-06 16:04:55):   SSE4.1 [Supported]
INFO    (2024-02-06 16:04:55):   AVX [Supported]
INFO    (2024-02-06 16:04:55):   AVX2 [Supported]
INFO    (2024-02-06 16:04:55):   PRFCHW [Supported]
INFO    (2024-02-06 16:04:55):  Done!
INFO    (2024-02-06 16:04:55): Loading settings from file 'settings.json'...
INFO    (2024-02-06 16:04:55):  Settings from file 'settings.json' loaded...
WARNING (2024-02-06 16:04:55): Failed to load settings file 'C:\Users\HOME\Documents\IOLITE Samples\settings.user.json'!
INFO    (2024-02-06 16:04:55): Initializing version 'development'...
INFO    (2024-02-06 16:04:55): Loading data packages...
ERROR   (2024-02-06 16:04:55):  The application crashed due to an unknown error.
```
Apparently the file `settings.user.json` file isn't loading because it doesn't exist in the folder. And also some unknown error. But thank you very much for answer!😃

**Comment by begla** [comment]
The settings file is optional (thus only a warning if it's not found). It looks like there is an issue reading one of the required data packages.

It is hard to guess what is going wrong, but you could start by re-downloading IOLITE and installing it in a different directory. It would be best if you begin testing with the minimal installation.

**Comment by Miroslav008** [comment]
> The settings file is optional (thus only a warning if it's not found). It looks like there is an issue reading one of the required data packages.
> 
> It is hard to guess what is going wrong, but you could start by re-downloading IOLITE and installing it in a different directory. It would be best if you begin testing with the minimal installation.

Ok, I'll try. Thanks a lot!)

**Comment by Miroslav008** [comment]
Nope, nothing works… I installed it in different ways, this way and that, but unfortunately nothing works. Can't I download an earlier version?

**Comment by begla** [comment]
> Nope, nothing works… I installed it in different ways, this way and that, but unfortunately nothing works. Can't I download an earlier version?

Did an earlier version launch successfully on your configuration? If so, I can provide you links to previous versions.

**Comment by Miroslav008** [comment]
> > Nope, nothing works… I installed it in different ways, this way and that, but unfortunately nothing works. Can't I download an earlier version?
> 
> Did an earlier version launch successfully on your configuration? If so, I can provide you links to previous versions.

No, I just found your engine and downloaded version 0.4.8. Downloading an older version is my last hope. Maybe try?

**Comment by Miroslav008** [comment]
Can you send me links to previous versions?

**Comment by begla** [comment]
https://media.missing-deadlines.com/iolite/builds/release/iolite-v0.4.0.zip

Try replacing the last digit of the version number with anything in `[0, 7]` to access other builds. Let me know if a build works. 🤞

**Comment by Miroslav008** [comment]
> https://media.missing-deadlines.com/iolite/builds/release/iolite-v0.4.0.zip
> 
> Try replacing the last digit of the version number with anything in `[0, 7]` to access other builds. Let me know if a build works. 🤞

I'll definitely give it to you! I understand about changing versions)

**Comment by Miroslav008** [comment]
Um... From version 0.4.5 and higher it crashes on startup with an error message. All other versions (from version 0.4.4 and older) crash without a message. And their logs do not say that there was any error. But they all crash. What's really interesting is that when I launched one of the older versions (0.4.2 or 0.4.3), IOLITE didn't crash, but it didn't boot either. Black screen. And Windows 10 said the program is not responding. So this is a problem on my part. I'll try to do everything I can.
Thank you very much for your support!) ☺🍀

**Comment by begla** [comment]
Any time. Please let me know when you find the root of the issue.

I will add more detailed crash reports in the future. When this is there, I can provide you with a specialized build to find out what is going on.

**Comment by Miroslav008** [comment]
> Any time. Please let me know when you find the root of the issue.
> 
> I will add more detailed crash reports in the future. I'll let you know when this is there and provide you with a specialized build to find out what is happening.

Nice. Good luck! I'm waiting for a specialized build.

**Comment by RedScipio** [comment]
Same bug here with V0.4.8

Video card: GeForce RTX 2060
Processor: Intel Core i5-136000K
RAM size: 32 GB
OS: Windows 10

If you want to do a debug session contact me

[application.log](https://github.com/MissingDeadlines/iolite/files/14440048/application.log)

**Comment by begla** [comment]
@RedScipio In your case, IOLITE is unable to read any file. Could you try reinstalling it to a different directory? This is potentially a path or permission-related issue.

**Comment by RedScipio** [comment]
Oh yes thanks !
Sorry for the false alarm

**Comment by begla** [comment]
@RedScipio I'm glad it worked out. 😄

**Comment by swanblack1** [comment]
exactly the same problem

**Comment by swanblack1** [comment]
Where is this file? I just dug through the entire folder and couldn't find it.

**Comment by swanblack1** [comment]
And here's the question: is it 32 bit or 64?Because as I looked it gave out 32 bit

**Comment by Miroslav008** [comment]
> Where is this file? I just dug through the entire folder and couldn't find it.

In _Documents\IOLITE_ iirc. Also, sorry for **_year late_**!
